cephprimarystorage/cephprimarystorage/cephagent.py

```
@@ -824,6 +826,18 @@ def cancel_storage_backup(self, req):
         cmd = jsonobject.loads(req[http.REQUEST_BODY])
         return self.imagestore_client.cancel_storage_backup(cmd)
 
+    @replyerror
+    def get_storage_backup_mode(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.12.2)

`http` may be undefined, or defined from star imports

(F405)